python/python/bystro/prs/preprocess_for_prs.py

```diff
@@ -125,38 +165,51 @@ def _preprocess_genetic_maps(map_directory_path: str) -> dict[int, list[int]]:
     return bin_mappings
 
 
-def filter_scores_by_p_value(scores: pd.DataFrame, p_value_threshold: float) -> pd.DataFrame:
-    """Filter to keep rows with P-values less than the specified threshold for C+T method."""
-    return scores[scores["P"] < p_value_threshold]
+def read_feather_in_chunks(file_path, columns=None, chunk_size=1000):
+    """Read a Feather file in chunks as pandas Dataframes."""
+    table = feather.read_table(file_path, columns=columns)
```

Comment:
Neat! This is way better than what I was doing before (which was lower level using dataset api)